README.md

```diff
@@ -46,7 +46,7 @@ The Helm Diff Plugin
 
 * Shows a diff explaining what a helm rollback would change:
     This fetches the currently deployed version of a release
-  and compares it to adeployed versions of a release, that you 
+  and compares it to deployed versions of a release, that you 
```

Comment:
initially, i was going to fix this to `a deployed`, but that didn't agree with `versions`. You could do:
```suggestion
  and compares it to a deployed version of a release, that you 
```
instead...

Reply:
It's better, but still doesn't make sense? Perhaps we should instead say...

```suggestion
  and compares it to the previously deployed versions of the release, that you 
```